server/src/db/peer.rs

```diff
@@ -338,6 +338,16 @@ impl DatabasePeer {
         Ok(peer_iter.collect::<Result<_, _>>()?)
     }
 
+    pub fn list_enabled(conn: &Connection) -> Result<Vec<Self>, ServerError> {
+        let mut stmt = conn.prepare_cached(&format!(
+            "SELECT {} FROM peers WHERE peers.is_redeemed = 1 AND peers.is_disabled = 0",
```

Comment:
Ideally we'd just do a `SELECT ip, name` but the deserialization machinery is all in place to select the whole peer so I just went with that for now for convenience. Performance likely wouldn't be a problem unless you have a _lot_ of peers.